src/source.c

```diff
@@ -197,7 +197,7 @@ dispatch_source_get_extended_data(dispatch_source_t ds,
 }
 
 void
-dispatch_source_merge_data(dispatch_source_t ds, unsigned long val)
+dispatch_source_merge_data(dispatch_source_t ds, uintptr_t val)
```

Comment:
This seems Linux specifically.

Reply:
This change was based on the [header declaration](https://github.com/apple/swift-corelibs-libdispatch/blob/13129d945443c8b6e8ceaa8f7ce922fc9751eebf/dispatch/source.h#L658), but the same change was again already fixed in #453.